extension-style/api/extension-style.api

```diff
@@ -3724,6 +3725,7 @@ public final class com/mapbox/maps/extension/style/sources/generated/GeoJsonSour
 }
 
 public final class com/mapbox/maps/extension/style/sources/generated/GeoJsonSource$Companion {
+	public final fun directSetterEnabled ()Z
```

Comment:
need to make this static method public to be visible for unit tests in annotation plugin